### PR TITLE
[GH-45] contribution guidelines and code of conduct

### PR DESCRIPTION
**Pull request by fwkoch**
Before merging, we should identify a better contact email for the code of conduct than info@gmggroup.org

See #45

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- confusion around the documentation
+- feature requests for the format
+- questions around the high-level goal of an open standard
+- suggestions around non-technical aspects, such as marketing and engagement
+- etc!
+
+If the question is non-technical, follow up may happen outside of
+GitHub, but this is the place to start.
+
+And, please, read and follow our [Code of Conduct](code_of_conduct.md).
```
Comment (rowanc1) [inline CONTRIBUTING.md]: This could be elevated rather than an after thought.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+the place for anyone to voice anything about OMF. Examples include:
+- detailed technical questions around implementation
+- problems encountered when attempting to support OMF
+- confusion around the documentation
+- feature requests for the format
+- questions around the high-level goal of an open standard
+- suggestions around non-technical aspects, such as marketing and engagement
+- etc!
+
+If the question is non-technical, follow up may happen outside of
```
Comment (rowanc1) [inline CONTRIBUTING.md:42]: a place to start.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+it's useful to know what people are working on. If your pull request
+does not have an existing issue, consider [creating an issue](#issues)
+first, just to add context and promote discussion.
+
+When working on your contribution, you may
+[fork](https://help.github.com/en/articles/fork-a-repo) the OMF repo to
+your personal or company GitHub organization and develop there.
+Alternatively, if you are interested in being identified as a contributor
+to the [GMG Group GitHub organization](https://github.com/gmggroup),
+reach out to [fwkoch](https://github.com/fwkoch) (note: this is
```
Comment (rowanc1) [inline CONTRIBUTING.md]: Should be a gmg contact email.

File: code_of_conduct.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+quick, friendly but clear word in an online message or in person can
+help resolve a misunderstanding and de-escalate things.
+
+An informal enforcement of this process may be inadequate if there is
+urgency, risk to someone, no one is comfortable speaking out, the
+offender is unresponsive, etc. In that case, please file a report
+by emailing [*info@gmggroup.org*](mailto:info@gmggroup.org). This will
+be kept anonymous and action will be taken.
+
+#### Attribution
```
Comment (rowanc1) [inline code_of_conduct.md:110]: Thanks for calling out these organizations!

**Review by rowanc1 — CHANGES_REQUESTED** [review]
This is really awesome and well written. Looking forward to community contributions.

**Review by rowanc1 — APPROVED** [review]

**Comment by dsanguinetti** [comment]
This looks good on the general conduct side of things. From a GMG perspective we probably need to add something covering anti-trust as well. In the mean-time I've asked about an e-mail address.

File: code_of_conduct.md (hunk 4 above)
Comment (dsanguinetti) [reply code_of_conduct.md:110]: After discussion with Heather, the e-mail address should be mine: dsanguinetti@gmggroup.org. Also, When referring to GMG it should be just GMG, not GMG group (this is coming from the communications folks...sorry!).
Thanks!
David

File: CONTRIBUTING.md (hunk 2 above)
Comment (banesullivan) [reply CONTRIBUTING.md:42]: Perhaps Slack or a discourse page would be appropriate?

**Review by banesullivan — COMMENTED** [review]
On:

> If the question is non-technical, follow up may happen outside of GitHub, but this is a place to start.

What about having a Slack channel dedicated to OMF somewhere? In the @OpenGeoVis slack workspace, I've created an `omfvtk` channel for using the [OMF-Viewer](https://github.com/OpenGeoVis/omfvtk)... It might be beneficial to have something like this for new users looking for answers to simple use cases. 

Or perhaps creating a [discrourse](https://www.discourse.org) community for OMF so that questions are archived and searchable? ([ParaView](https://discourse.paraview.org) and [VTK](https://discourse.vtk.org) have really good examples of this)

**Comment by dsanguinetti** [comment]
Hi Bane,
    We've tried Slack with the AI group in GMG, and the conclusion so
      far is that for the asynchronous sort of communication that we get
      across time zones it is better to use Basecamp or e-mail. That
      being said, we are investigating other platforms as well because
      we want to have a venue for a community of interest to share
      ideas, exactly as you've described.
    David

**Pull request merged**